examples/fluids/qfunctions/euler-outflow.h

```diff
@@ -1,117 +0,0 @@
-// Copyright (c) 2017, Lawrence Livermore National Security, LLC. Produced at
-// the Lawrence Livermore National Laboratory. LLNL-CODE-734707. All Rights
-// reserved. See files LICENSE and NOTICE for details.
-//
-// This file is part of CEED, a collection of benchmarks, miniapps, software
-// libraries and APIs for efficient high-order finite element and spectral
-// element discretizations for exascale applications. For more information and
-// source code availability see http://github.com/ceed.
-//
-// The CEED research is supported by the Exascale Computing Project 17-SC-20-SC,
-// a collaborative effort of two U.S. Department of Energy organizations (Office
-// of Science and the National Nuclear Security Administration) responsible for
-// the planning and preparation of a capable exascale ecosystem, including
-// software, applications, hardware, advanced system engineering and early
-// testbed platforms, in support of the nation's exascale computing imperative.
-
-/// @file
-/// Euler outflow BC
-
-#ifndef euler_outflow_h
-#define euler_outflow_h
-
-#include <math.h>
-
-#ifndef euler_context_struct
```

Comment:
To make this work the way you want, you would need to put the struct in a third file and include it in both qfunction headers.

Reply:
Thanks @jeremylt for the hint. I will go ahead and merge this since we don't need to split the headers right now. We can split them when we add other problems that use `euler_outflow`.